### ISSUE TITLE
Frostbolt doesn't sync mana cost between ranks

### ISSUE THREAD

**Issue by XionDemise** [chromiecraft/chromiecraft#798]
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [X] enUS
- [ ] other (specify)

##### FACTION

- [X] Alliance
- [ ] Horde

##### CONTENT PHASE:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

it seems a few of the frostbolt ranks are different and may cause an advantage in some cases, It isn't a classic client either.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

They're all supposed to cost the same amount of mana as of 3.3.5

##### STEPS TO REPRODUCE THE PROBLEM:

1. create a mage 
2. lvl it up to 80
3. check all frostbolt ranks
4.

##### EXTRA NOTES:

Type here.

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/617a935ee469641f3689061bb390158040b5d8d7

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by pop-pig** [comment]
On the DB if you filter "Frostbolt" https://wowgaming.altervista.org/aowow/?spells=7.8.6
Ranks 1, 2, 3 and 4 have reduced base mana cost. From rank 5 on, it's 11% per cast.
I cross checked on wotlkdb.com (link: https://wotlkdb.com/?spells=7.8.6 ), it's the same.
I have no idea how it really was back then though.

**Comment by pop-pig** [comment]
I've been looking up the WoW wiki, and here I've found a nice table: https://wowpedia.fandom.com/wiki/Frostbolt?oldid=2406668
This is pulled from the history of the Frostbolt page, last revision before Cata pre-patch (it's the October 7th 2010 version of the page, Cataclysm pre-patch was released on October 12th ; also previous version of the page have the same values).
Mana cost percentage seem to be coherent.
I think we can close the issue?
